src/drop.ts

```diff
@@ -18,18 +23,72 @@
  * - Drops currently cannot be removed by other abilities.
  * - Drops are essentially permanent although this may change in the future.
  */
+export type DropType =
```

Comment:
Maintaining manual list = awful
I've opened #2294 to deal with this in the future.